tests/sqllogictests/suites/query/functions/02_0012_function_datetimes.test

```diff
@@ -901,6 +901,41 @@ select date_add(QUARTER, 1, to_date('2018-01-02'))
 ----
 2018-04-02
 
+query T
+SELECT DATE_ADD(WEEK, 1, TODAY())=DATE_ADD('day', 7, TODAY())
+----
+1
+
+query T
+SELECT DATE_DIFF(YEAR, to_date('2000-01-02'), to_date('2024-01-01'))
+----
+23
+
+query T
+SELECT DATE_DIFF(YEAR, to_date('2012-02-29'), to_date('2013-02-28'))
+----
+1
```

Comment:
consider test select DATE_DIFF(YEAR, TO_TIMESTAMP('2023-12-31 09:38:18.165575'), TO_TIMESTAMP('2024-01-01 21:01:35.423179')) the result should be 1.

Reply:
Also need to add some tests where the sign is negative